shipyard_blueprints/hubspot/shipyard_hubspot/hubspot_utils.py

```diff
@@ -159,28 +159,28 @@ def column_to_hubspot(
     :param column_type: The column type: HUBSPOT_OBJECT_ID, HUBSPOT_ALTERNATE_ID
     :return: importRequest Column Mapping
     """
-    if column_type:
-        column_type = column_type.strip().upper()
-        if column_type not in {"HUBSPOT_OBJECT_ID", "HUBSPOT_ALTERNATE_ID"}:
-            raise ExitCodeException(
-                "Invalid column type. Column type is used to specify that a column contains a unique identifier "
-                "property Please choose between HUBSPOT_OBJECT_ID, HUBSPOT_ALTERNATE_ID",
-                Crm.EXIT_CODE_INVALID_INPUT,
-            )
-
-        return {
-            "columnObjectTypeId": column_object_type_id,
-            "columnName": csv_column_name,
-            "propertyName": hubspot_property_name,
-            "columnType": column_type,
-        }
-    else:
+    if not column_type:
         return {
             "columnObjectTypeId": column_object_type_id,
             "columnName": csv_column_name,
             "propertyName": hubspot_property_name,
         }
 
+    column_type = column_type.strip().upper()
+    if column_type not in {"HUBSPOT_OBJECT_ID", "HUBSPOT_ALTERNATE_ID"}:
```

Comment:
Could these be declared as a global variable? Something where they could be referenced/edited more easily if more column types are needed eventually.